src/PortToDocs/src/libraries/XmlHelper.cs

```diff
@@ -33,18 +33,30 @@ internal class XmlHelper
             { "></see>",        " />" }
         };
 
+        private static readonly Dictionary<string, string> _replaceableNormalElementRegexPatterns = new Dictionary<string, string>
+        {
+            // Replace primitives: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/built-in-types
+            { @"\<(see|seealso){1} cref\=""bool""[ ]*\/\>",    "<see cref=\"T:System.Boolean\" />" },
```

Comment:
If the `<seealso>` tag is a child of the `<docs>` tag, then it should remain as a `<seealso>` tag, which adds an entry to the See Also section of the docs. Will this convert those to `<see>` tags?

Reply:
No, those would remain untouched. This dictionary is only used to convert the see to seealsos when found within a summary, a returns, or a description of a typeparam/param.